phasme/graphics.py

```diff
@@ -1,27 +1,31 @@
-import matplotlib.pyplot as plt
-import networkx as nx
-import numpy as np
-import os
-
 """Module containing general routines for graphics generation
 using pandas/matplotlib.
 
 Used by routines to produce loads of visualizations.
 
 """
 
+import matplotlib.pyplot as plt
+import networkx as nx
+import numpy as np
+import os
+
+
 def make_all(graph, outdir):
-    make_degree_distrib(graph, outdir)
-    make_coef_distrib(graph, outdir)
-    make_coef_distrib_stacked(graph, outdir)
+    outfile = make_degree_distrib(graph, outdir)
+    yield outfile, make_degree_distrib.__doc__.splitlines(False)[0]
```

Comment:
You should refactor all `.__doc__.splitlines(False)[0]` in a `get_description` function. Simpler to read.